src/docs/reference/modules/AutoSetup/README.md

```diff
@@ -70,9 +70,26 @@ Auto-Setup parameters are defined in appsettings.json. Example excerpt:
     `/autosetup` - trigger installation of the Root tenant.
     `/mytenant/autosetup` - auto-install mytenant.
 
-### Environment Variables
+### User Secrets and Environment Variables
+
+Since JSON configuration contains admin-sensitive information, it is recommended to use user secrets or environment variables instead.
```

Comment:
Since something like a dev config for Auto Setup for the whole team is neither sensitive nor should be left out of the repo:

```suggestion
If your JSON configuration contains sensitive information, or you don't want to commit it to the repository (because e.g. Auto Setup is not utilized by the whole development team), it is recommended to use user secrets or environment variables instead.
```